scriptworker/constants.py

```diff
@@ -212,9 +212,13 @@
             ),
             'beta': (
                 "/releases/mozilla-beta",
+                # TODO remove jamun once pushapk is landed on beta
+                "/projects/jamun",
             ),
             'aurora': (
                 "/releases/mozilla-aurora",
+                # TODO remove date once pushapk is landed on aurora
+                "/projects/date",
```

Comment:
I was intentionally leaving date and jamun out of 'beta' and 'aurora', since pushing to google play from staging branches could be bad.  If you need this for testing, we can keep it, but please make very sure that we don't accidentally update the play store.

This also puts us at risk for updating balrog from the wrong branch.  Maybe we should create a new 'auroratest' and 'betatest' that allows for date and jamun, and point the googleplay scopes at those?  That would avoid opening up balrog to pushes from date and jamun.